CONTRIBUTING.md

```diff
@@ -3,8 +3,8 @@
 ## Setup developer environment
 
 ```bash
-git clone git@github.com:kinvolk/lokoctl.git
```

Comment:
regarding CONTRIBUTING.md file move, from GitHub perspective, it's fine to have this file in `docs` directory, it will still be detected properly.

Reply:
ah, cool, I'll leave it there then.